articles/application-gateway/application-gateway-backend-health-troubleshooting.md

```diff
@@ -426,7 +426,7 @@ This behavior can occur for one or more of the following reasons:
 
     b.	Check whether there's any UDR configured. If there is, search for the resource on the search bar or under **All resources**.
 
-    c.	Check whether there are any default routes (0.0.0.0/0) with the next hop not set as **Internet**. If the setting is either **Virtual Appliance** or **Virtual Network Gateway**, you must make sure that your virtual appliance or the on-premises device can properly route the packet back to the internet destination without modifying the packet.
+    c.	Check whether there are any default routes (0.0.0.0/0) with the next hop not set as **Internet**. If the setting is either **Virtual Appliance** or **Virtual Network Gateway**, you must make sure that your virtual appliance or the on-premises device can properly route the packet back to the internet destination without modifying the packet. In this scenario, it is possible to see the backend responding with a 200 response and traffic still routing to the Application Gateway without issue.
```

Comment:
@bmattmcneilly013 , thanks for the contribution! Can you let me know why this clarification is needed? I'm not quite getting what this sentence tries to convey.

Reply:
I added the clarification because I had two cases in the last 6 months or so where the backend responded with 200s per web logs, but they still had unknown backend health because they had UDRs forcing the health probes through the NVA. The customers were concerned that meant connection was possibly unstable because they understood that anything other than a healthy status put his traffic at risk. In the end, they still chose to route probes through the NVA because we were able to affirm that their configuration wasn't unstable, but we could have avoided ticket creation by specifying expected behavior in their configuration.

Reply:
@bmattmcneilly013 , gotcha. In that case, I think it needs a bit more clarification. Can you elaborate that line a bit please?

Reply:
I think this is more clear:

In the event your probes are routed through a virtual appliance and modified, it is possible the backend resource will respond to probes with a 200 response, and traffic will route to the Application Gateway without issue, though the Application Gateway health status will display as "unknown".